### PR TITLE
fix some unit test cases(ink_well_test.dart) bug

### PR DESCRIPTION
**Pull request by xu-baolin**
## Description

This change fixes some bugs in `ink_well_test.dart`, see file for details.

## Tests

I added the following tests:

Fix some test cases.

### CODE REVIEW

File: packages/flutter/test/material/ink_well_test.dart [hunk 1]
```diff
@@ -350,31 +350,29 @@ void main() {
       child: Directionality(
         textDirection: TextDirection.ltr,
         child: Center(
-          child: Focus(
-            focusNode: focusNode,
```
Comment (xu-baolin) [inline packages/flutter/test/material/ink_well_test.dart:354]: Request this node focus can not let the `InkWell` focused, the `InkWell` has its own `Focus` and node.

File: packages/flutter/test/material/ink_well_test.dart [hunk 2]
```diff
@@ -185,7 +185,7 @@ void main() {
     );
     await tester.pumpAndSettle();
     final RenderObject inkFeatures = tester.allRenderObjects.firstWhere((RenderObject object) => object.runtimeType.toString() == '_RenderInkFeatures');
-    expect(inkFeatures, paintsExactlyCountTimes(#rect, 0));
```
Comment (xu-baolin) [inline packages/flutter/test/material/ink_well_test.dart:188]: Its a typo of `#drawRect`.

**Comment by xu-baolin** [comment]
@gspencergoog  Hi, please review. thanks very much

**Review by gspencergoog — APPROVED** [review]
32384589-a60f0e74-c078-11e7-9bc1-e5b5287aea9d

**Pull request merged**